### ISSUE TITLE
running GeMoMa in a container - 

### ISSUE THREAD

**Issue by colindaven** [Jstacs/Jstacs#60]
Hi,
I'm trying to run GeMoMa in a singularity container built from a Dockerfile.

The `container is read only ` so at runtime I am getting this error. Is there a chance the `ini.xml `could be written to the current directory as a fallback ? Is the default file important and what happens if it can't be written and later read ?

Thanks,
Colin

```
Default values could not be written to /opt/micromamba/share/gemoma-1.9-0/GeMoMa.ini.xml
java.io.FileNotFoundException: /opt/micromamba/share/gemoma-1.9-0/GeMoMa.ini.xml (No such file or directory)
        at java.base/java.io.FileInputStream.open0(Native Method)
        at java.base/java.io.FileInputStream.open(FileInputStream.java:219)
        at java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)
        at java.base/java.io.FileReader.<init>(FileReader.java:75)
        at de.jstacs.io.FileManager.readFile(FileManager.java:207)
        at projects.gemoma.GeMoMaPipeline.<clinit>(GeMoMaPipeline.java:113)
        at projects.gemoma.GeMoMa.main(GeMoMa.java:344)

```

**Comment by colindaven** [comment]
Hi Jens, thanks for the quick reply. 

I've tried to rerun without a container but it is proving difficult to get the file `GeMoMa.ini.xml` without installing all the dependencies such as mmseqs etc. I have already installed Gemoma and Java etc.

Is there any other place I can get hold of this (a gist, repo or something?) ? Perhaps a default version could be included in the repo under tests etc, since other `xml` are present. To be usable in containers, which are popular these days as you know,  then a default version of the file should likely be included in the conda distribution along with the JAR file.

Thanks! 
Colin
